tests/ui/associated-type-bounds/return-type-notation/bad-inputs-and-output.rs

```diff
@@ -13,7 +13,7 @@ fn foo<T: Trait<method(i32): Send>>() {}
 fn bar<T: Trait<method() -> (): Send>>() {}
 //~^ ERROR return type not allowed with return type notation
```

Comment:
Why doesn't this complain about missing `..`?

Reply:
order of error reporting, we first error about `->` then error about missing `..`. I could make them report in some other order, but 🤷

Reply:
Yeah, not worth it unless super trivial to maintain, even after future stabilization.